doc/RELEASES.md

```diff
@@ -0,0 +1,30 @@
+# Releasing the Dependency Track Maven Plugin
+
+To initiate a new release, raise a Pull Request to the master branch which contains the requested change(s) and sets the 
+POM version number to the next release version.
+
+This project follows semantic versioning principles laid out at [SemVer](https://semver.org/).  As such please follow 
+these guidelines when setting the project version number:
+* Major - Reserved for major releases as defined by the project team or when changes are made to the plugin interface 
+that are not backwards compatible with the previous version.
+* Minor - Backwards compatible changes is to the interface, e.g. a new goal, a new configuration
```

Comment:
extra "is" on '* Minor - Backwards compatible changes is to the interface, e.g. a new goal, a new configuration'

Reply:
Cheers - PR updated.